README.md

```diff
@@ -74,8 +74,11 @@ Support for offline maps is available by *"side loading"* the required map tiles
 ```
 
 ## Location features
+To enable location features in an **Android** application:
 
-To enable location features in an iOS application:
+You need to declare the `ACCESS_COARSE_LOCATION` or `ACCESS_FINE_LOCATION` permission in the AndroidManifest.xml and starting from Android API level 23 also request it at runtime. The plugin does not handle this for you. The example app uses the flutter ['location' plugin](https://pub.dev/packages/location) for this. 
```

Comment:
The `location` plugin is also required for iOS right? If so we should maybe put this remark at the beginning (or end) of this section. In the current form it seems that the plugin is not required for iOS.

Reply:
I'm really not experienced at iOS development, but I thought it was enough to include the keys in Info.plist and the system would automatically ask the user for permission at runtime if you use location features. I agree that we should rephrase it if that's not the case.

Reply:
I've just tested it by re-installing the app and navigating to map_ui. You're absolutely right that no separate plugin is needed to handle the permissions.